calibrations/tpc/gemcurrents/trip_history.py

```
@@ -0,0 +1,1048 @@
+    # ==============================================================
+
+    def _query_prometheus(self, variable_name):
+
+        params = {
+            "query": variable_name,
+            "start": self.begin_time,
+            "end": self.end_time,
+            "step": self.PROMETHEUS_STEP
+        }
+
+        response = requests.get(
```

Ruff (0.16.3)

[error] Probable use of `requests` call without timeout

(S113)